site/docs/v1/tech/tutorials/integrate-angular.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: doc
+title: Integrate Your Angular Application With FusionAuth
+description: Integrate your Angular application with FusionAuth
+navcategory: getting-started
+prerequisites: nodejs
+technology: Angular
+language: TypeScript
+---
+
+{% include docs/integration/_intro.md %}
```

Comment:
I think we'll have to change `_intro.md` to allow passing a variable to change _"how to integrate **a {{page.technology}}** application"_ to _"how to integrate an **{{page.technology}}** application"_

> In this article, you are going to learn how to integrate **a Angular** application with FusionAuth.

Reply:
I added a small if statement, which I think does the job. Don't have any other prefix issues.